DocumentService/Controllers/DocumentsController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace DocumentService.Controllers
+{
+    [Route("api/")]
+    [ApiController]
+    public class DocumentsController : ControllerBase
+    {
+
+        public DocumentsController()
+        {
+
+        }
+
+        /// <summary>
+        /// Upload a document.
+        /// </summary>
+        /// <param name="CorrelationId">Correlation identifier of the operation.</param>
+        /// <param name="UserName">Azure AD identifier of the user uploading the document.</param>
+        /// <param name="FileName">File name of the document.</param>
```

Comment:
Flagging that I added a new parameter FileSize after FileName. See design document for explanation why I added it and see if you agree :)